addons/static_radios/edenAttributes.hpp

```diff
@@ -30,8 +30,8 @@ class Attributes {
         typeName = "NUMBER";
     };
     class staticRadioSpeaker {
-        displayName = "Speaker enabled"; // Name assigned to UI control class Title
-        tooltip = "Whether the Radios Speaker is enabled. Doesn't work on Static LR Radio backpacks yet (#1169).";
```

Comment:
The second part of that is a comment that should be marked as #TODO and does not belong into the string or any translations.

Reply:
I don't really understand.
Are you mean every I change string if that string have comment.
I must add #TODO into comment prefix.

Like: 
Original: // Some comment........
Fix: //#TODO Some comment......

Is this your mean ?

Reply:
The text to be translated is `Whether the Radios Speaker is enabled.`
And just put the `Doesn't work on Static LR Radio backpacks yet (#1169).` as a `//#TODO Doesn't work on Static LR Radio backpacks yet (#1169).` behind the semicolon on that line.

Reply:
Ha, I get it. Roger that.